app-modules/ai/src/Filament/Pages/ManageAiIntegratedAssistantSettings.php

```diff
@@ -86,12 +86,12 @@ public function form(Form $form): Form
                     ->helperText('Used for general purposes like generating content when an assistant is not being used.')
                     ->required(),
             ])
-            ->disabled(! auth()->user()->can('product_admin.*.update'));
+            ->disabled(! auth()->user()->can('saas_global_admin.*.update'));
```

Comment:
This should be checking if they are a global admin rather than a permission. There is a helper for this. `$user->isSuperAdmin()`